packages/.github/workflows/dart.yml

```diff
@@ -0,0 +1,796 @@
+# Created with package:mono_repo v6.5.0
```

Comment:
This is placed wrongly here

Reply:
oh I thought `mono_repo generate` will also run relative to the project root :eyes:

Reply:
Apparently not, maybe it's a bug? Doesn't seem to be too useful for me to not have it relative to the project